docs/resources/modelarts_notebook_mount_storage.md

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "AI Development Platform (ModelArts)"
+---
+
+# huaweicloud_modelarts_notebook_mount_storage
+
+Manage storages mounted to the notebook resource within HuaweiCloud. A maximum of 10 storages can be mounted.
+
+## Example Usage
+
+```hcl
+variable "notebook_id" {}
+variable "uri_parallel_obs" {}
+
+resource "huaweicloud_modelarts_notebook_mount_storage" "test" {
+  notebook_id = var.notebook_id
+  uri         = var.uri_parallel_obs
+  mount_path  = "/data/test/"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the resource. If omitted, the
+  provider-level region will be used. Changing this parameter will create a new resource.
+
+* `notebook_id` - (Required, String, ForceNew) Specifies ID of notebook which the storage be mounted to.
+ Changing this parameter will create a new resource.
+
+* `uri` - (Required, String, ForceNew) Specifies the uri of Parallel File System (PFS) or its folders in OBS.
+ The format is : `obs://obs-bucket/folder/`. Changing this parameter will create a new resource.
+
+* `mount_path` - (Required, String) Specifies the local mount directory. Only the sub directory of `/data/`can be
+ mounted. The format is : `/data/dir1/`. Changing this parameter will create a new resource.
+
+* `type` - (Optional, String, ForceNew) Specifies the type of storage system. The options are as follows:
+  - **OBSFS**: Parallel File System (PFS) of OBS.
+  
+  Default value is `OBSFS`,
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID in format of **notebook_id/mount_id**. It is composed of the ID of notebook and mount ID,
+ separated by a slash.
+
+* `mount_id` - The mount ID.
```

Comment:
suggest to use mount_id as the resource id.

Reply:
This format is more convenient for users.